docs/configuration/protocols/bfd.rst

```diff
@@ -9,6 +9,11 @@ BFD
 :abbr:`BFD (Bidirectional Forwarding Detection)` is described and extended by
 the following RFCs: :rfc:`5880`, :rfc:`5881` and :rfc:`5883`.
 
+In the age of very fast networks, a second of unreachability can means millions of packet loss. The idea behind BFD is to detect very quickly when a peer is down and take action extremely fast. 
+
+For example without BFD, when a BGP or OSPF peer is unreachable but if the interface is still up it needs to wait for the timeout to expires to conludes that a peer is unreachable.
+
+BFD uses small UDP packet very quickly to ensures that the peer is still alive.
```

Comment:
Please run the patch through grammarly.com or similar. The content is good, but the number of grammar mistakes seemingly coming from mere inattention is overwhelming.

Reply:
Ha ha Sorry, English is not my mother tongue. I am doing my best though ;)

Reply:
That's why I suggest that you should use grammarl.com

Reply:
Wel, I did follow your suggestion. But I guess that's not enough.

Reply:
I don't know what you did. If you simply accept its suggestions, most of the problems will be gone.